h/config.py

```diff
@@ -143,6 +143,11 @@ def configure(environ=None, settings=None):
     es_logger = logging.getLogger("elasticsearch")
     es_logger.addFilter(ExceptionFilter((("ReadTimeoutError", "WARNING"),)))
 
+    # Add API version settings
+    # These cannot be overridden by env
+    settings["api.versions"] = ["v1", "v2"]
+    settings["api.version.current"] = "v1"
```

Comment:
I wonder if `"api.version.default"` would be a better name? Since I think this is the default version that you get if you don't request a particular version in an `Accept` header.

Do I understand correctly that these two settings cannot be changed either by INI files or by envvars? So if we wanted to change either one of these settings, we'd have to change the Python code and redeploy? In that case I think these should actually be `CONSTANTS` not settings.

If the suggestion is amenable here's some ideas for where to put constants:

1. If they're only used within `h.views.api` make them private constants somewhere in that package:

   1. If only needed in one `h/views/api/*.py` module then make them `_CONSTANTS` in that module
   1. If they're needed throughout the `h.views.api` package, put them as private `_CONSTANTS` in `h/views/api/__init__.py` or, if that would cause a circular import, in `h/views/api/_constants.py` (and then `__init__.py` and any other modules that need them import them from `_constants.py`)

1. Or if they're potentially needed throughout the entire app then make them public `CONSTANTS` as part of the `h.views.api` public API. They would go in `h/views/__init__.py` unless they had to be moved into `h/views/_constants.py` to avoid circular imports. Either way `__init__.py` would add them to its `__all__` in order to make them part of its public API.

   Any code throughout the app could then just do `from h.views.api import API_VERSION_CURRENT`.